openTSNE/nearest_neighbors.py

```diff
@@ -217,8 +217,9 @@ class Annoy(KNNIndex):
         "taxicab",
     ]
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, n_trees=50, **kwargs):
```

Comment:
See the NNDescent comment below. Ideally, this would come through a `knn_kwargs` parameter containing a dictionary of parameters, so as to avoid having to keep these classes in sync with the underlying Annoy implementation.

Reply:
I have now reworked this a bit.  Not sure if the current state is satisfying because we now pass in `n_trees` as an entry in the `knn_kwargs` dict, but the value actually goes into `index.build(...)`.